src/utils/package-json.ts

```diff
@@ -36,9 +36,9 @@ const cdsChanges = {
 
 const scaffoldProjectPackageJson: PackageJsonChange = {
   scripts: {
-    'deploy': 'npm run ci-build && npm run ci-package && cf push',
+    'deploy': 'npm run build && sap-cloud-sdk package && cf push',
     'ci-build': 'npm run build',
-    'ci-package': 'sap-cloud-sdk package',
+    'ci-package': 'sap-cloud-sdk package --ci',
```

Comment:
So the following use case builds **once**:
- run `deploy`
- run `ci-build` + `ci-package`

Every use case above makes sense.
However, if I build/package/test my app in pipeline with `ci-xxx` commands, how should I deploy?
Should we add a `ci-deploy` command, since the running with `deploy` will build twice.

Reply:
Good point. I also thought about that, but the `ci-` scripts are invoked by our pipeline and I think this is done in multiple steps / stages, right @fwilhe? There is no `ci-deploy` script instead.

Reply:
Discussed, this should be aligned with the pipeline, which should not block this PR.